pyqt5-gui/main_menu.py

```diff
@@ -243,42 +243,85 @@ def open_func(subcat_name):
         """OPEN EDIT ACCOUNT LAYOUT/REPLACE CATEGORIES LAYOUT"""
```

Comment:
Fix phone number label and textbox appearance in the for loop

Reply:
Fixed with https://github.com/Stake-And-Rope/market-app/commit/c937fae0106bc61c90d87397c5e630778f580662

Details about the fix are described in #42